features/roomlist/impl/src/main/kotlin/io/element/android/features/roomlist/impl/RoomListState.kt

```diff
@@ -49,6 +49,8 @@ data class RoomListState(
             val roomId: RoomId,
             val roomName: String,
             val isDm: Boolean,
+            val markAsUnreadFeatureFlagEnabled: Boolean,
```

Comment:
I think we should not reflect on the ui state if it's a feature flag or anything else, so would just rename for `markAsUnreadEnabled ` ?

Reply:
Good point, I will think about it next time. I do not believe that this flag will live long, so we will remove this code anyway.